README.md

```diff
@@ -134,13 +134,13 @@ You can find all the available options in the
 
 Once your bot has been deployed, add it to your Campfire instance. In the bot
 section of the Campfire UI you can give your bot a name, and set its webhook URL
-to be an endpoint of your bot (like `http://mybot.example.com/cats`). Then try
+to be an endpoint of your bot (like `https://mybot.example.com/cats`). Then try
```

Comment:
I think we should leave this as `http` since the sample bot instructions don't cover setting up SSL.

Of course for a production bot you'd want to use `https`, but when experimenting with the sample bot I think plain `http` is easier. We could add a note to clarify that, too.